static/src/javascripts/projects/common/modules/article/spacefinder.js

```diff
@@ -120,25 +117,36 @@ define([
         };
     }
 
-    function _enforceRules(slots, rules, bodyHeight) {
+    function _enforceRules(slots, rules, bodyTop, bodyHeight) {
         var filtered;
 
+        // enforce absoluteMinAbove rule
+        if (rules.absoluteMinAbove > 0) {
+            filtered = Promise.resolve(filter(slots, function (slot) {
+                return bodyTop + slot.top >= rules.absoluteMinAbove;
+            }));
+        }
+
         // enforce minAbove and minBelow rules
-        filtered = Promise.resolve(filter(slots, function (slot) {
-            var farEnoughFromTopOfBody = slot.top >= rules.minAbove,
-                farEnoughFromBottomOfBody = slot.top + rules.minBelow <= bodyHeight;
-            return farEnoughFromTopOfBody && farEnoughFromBottomOfBody;
-        }));
+        filtered = filtered.then(function (slots) {
+            return filter(slots, function (slot) {
+                var farEnoughFromTopOfBody = slot.top >= rules.minAbove,
+                    farEnoughFromBottomOfBody = slot.top + rules.minBelow <= bodyHeight;
```

Comment:
Let's use multiple vars as we decided lately. In this case I would actually find it really helpful as they are quite long lines to read.